cypress/support/fragments/data_import/mapping_profiles/mappingProfileDetails.js

```diff
@@ -57,6 +59,14 @@ export default {
     ]);
   },
 
+  deleteMappingProfile:(name) => {
+    cy.do([
+      Pane({ id:'full-screen-view' }).find(actionsButton).click(),
+      deleteButton.click(),
+      Modal(`Delete "${name}" field mapping profile?`).find(deleteButton.click())
```

Comment:
JFYI Also you can use 

Modal(including(name)).find(deleteButton.click())

It is just shorter, but you aproach is ok)

Reply:
got it, thanks